sources/transaction.move

```diff
@@ -99,13 +99,47 @@ public fun p2pkh_address(output: &Output): vector<u8> {
 		script[23] == OP_EQUALVERIFY &&
 		script[24] == OP_CHECKSIG
     ) {
-		return slice(script, 3, 23)
+		return true
 	};
-    vector[]
+    false
 }
 
-public fun amount(output: &Output): u256 {
-    output.amount
+public fun is_op_return(output: &Output): bool {
+    let script = output.script_pubkey;
+    return script.length() > 0 && script[0] == OP_RETURN
+}
+
+public fun p2pkh_address(output: &Output): vector<u8> {
+    // TODO: we support P2PKH and P2PWKH now.
+    // We will and more script after.
+    // and the script must return error if we don't support standard script
+
+    let script = output.script_pubkey;
+	return slice(script, 3, 23)
+}
+
+/// Return message after OP_RETURN
```

Comment:
after OP_RETURN? its a little bit unclear. Could you add a more descirptive comment here?